qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -69,7 +69,7 @@ class AuthServiceProxy(object):
 
     def __init__(self, service_url, service_name=None, timeout=HTTP_TIMEOUT, connection=None):
         self.__service_url = service_url
-        self.__service_name = service_name
+        self._service_name = service_name
```

Comment:
@laanwj note here that unfortunately I wasn't able to completely avoid changing `authproxy.py`: in order to use composition over inheritance in `coverage.AuthServiceProxyWrapper` (which I think is preferable), we have to make this attribute un-name-mangled, i.e. accessible outside of the class itself.